voluncheer/.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 100
 exclude = */migrations/*
-ignore = E203
+ignore = E203,E402
```

Comment:
We should not ignore this across the project. If you absolutely must use it, use it locally for single exceptions.